Docs/sphinx/Algorithms.rst

```diff
@@ -512,7 +512,7 @@ The mixture-averaged transport coefficients discussed above (:math:`\mu`, :math:
 
 The following choices are currently implemented in `PeleC`
 
-* The viscosity, :math:`\mu`, is estimated based <something>
+* The viscosity, :math:`\mu`, is estimated based 
```

Comment:
Add the equation, which has a similar form as the one below for thermal conductivity, but with alpha = 6 (see code: https://github.com/AMReX-Combustion/PelePhysics/blob/05dc43de5239a0f189ec73fd6a19588dffd0b99a/Transport/Simple.H#L332)

Reply:
Added the equation for viscosity.